### DISCUSSION TITLE
How to get DisplayObject's bounds based on root stage (a.k.a screen)?

### DISCUSSION THREAD

**Discussion by humbleCoder1990** [pixijs/pixijs#7790] [Q&A] [answered]
I want to get absolute position of DisplayObject based on browser screen, not parent container.
Because I'm trying to layer DOM element on It.
The problem is I can't get right 'x, y".
Thanks in advance.

**Comment by ivanpopelyshev** [comment, marked as answer]
`getBounds()` ?
